lib_test/releases.ml

```diff
@@ -1,32 +0,0 @@
-open Printf
```

Comment:
intentionally deleting the releases binary here?

Reply:
Yes that is deliberate for 2 reasons:
 1. The `hub` command can test the same code minor formatting differences aside
     eg ` dune exec --  hub/hub.exe release --owner mirage --repo ocaml-github`
 2. This binary wasn't listed in the dune file so it wasn't being compiled.

In future I'd like to have `lib_test` executables bundled into the `hub` command with some `mdx` tests wrapping them to provide some basic assurances.